Frends.AzureBlobStorage.ReadBlob/Frends.AzureBlobStorage.ReadBlob/Frends.AzureBlobStorage.ReadBlob.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
 	<TargetFrameworks>net6.0</TargetFrameworks>
-	<Version>1.0.0</Version>
+	<Version>100.0.0</Version>
```

Comment:
I believe this is little mistake.

Reply:
Forgot to revert this 1 after tenant testing.